index.js

```diff
@@ -101,6 +101,9 @@ class BitbucketScm extends Scm {
 
         return this.breaker.runCommand(options)
             .then((response) => {
+                if (response.statusCode === 404) {
```

Comment:
For Github, it returns a 404 regardless if the the repository is private or the repository is missing.

What status code does Bitbucket return when it's a private repository?